### ISSUE TITLE
Marble bonus

### ISSUE THREAD

**Issue by Alkeif** [yairm210/Unciv#1818]
**Describe the bug**
Marble does not provide its bonus production to wonders. 

**Expected behavior**
Marble provides +15% production to all wonders.

**Additional context**
The bonus was limited to ancient and classical wonders in BNW but in vanilla and G&K it is for all wonders

**Comment by yairm210** [comment]
![image](https://user-images.githubusercontent.com/8366208/73596665-d4d7a000-452c-11ea-8504-50a27fb5e288.png)

Um... it does
What data are you going on?

**Comment by Alkeif** [comment]
Oh sorry. Let me rephrase it. In game description of marble does not mention such effect.

**Comment by yairm210** [comment]
Good point!

**Comment by Alkeif** [comment]
I would also like to add that I never knew the railroad bonus exists until I saw your post above. Nice to know! I think it should also be mentioned in its in game description.

**Comment by yairm210** [comment]
Okay, for both of the above - where do you think this information should be displayed in-game? In the Civilopedia is a start, but not everyone checks that.

**Comment by Alkeif** [comment]
That's a good start. That's the only place I could think of also.Other things I saw that needs revision in the civilopedia are:
Cultural victory policies from 4 to 5

Use of plural words for effects that are singular(giving only +1 effect)
Ex. +1 Engineer specialist slot (not slots)
+1 Great Merchant point (not points)

**Comment by alextfish** [comment]
The Railroad technology in the tech tree could also mention the Railroad production bonus. And somewhere, either the tech or the terrain improvement, should mention that the production bonus is also conveyed via Harbors. 

And yes, I think simply mentioning this apparent 15% wonder production bonus in the Civilopedia for marble would be great. (I had no idea about it either.)
